sympy/physics/control/lti.py

```diff
@@ -2392,6 +2439,40 @@ def var(self):
         """
         return self.sys1.var
 
+    @property
+    def num(self):
+        """
+        Returns the numerator of the Feedback equivalent Transfer Function.
+
+        Returns:
+        - Expression: Numerator expression of the Feedback Transfer Function.
```

Comment:
I would say the first line is self-explanatory. Remove this.
Same for den.

Reply:
I've removed the extra lines.